frontend/packages/dev-console/src/components/import/advanced/ProgressiveList.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import FormSectionDivider from '../section/FormSectionDivider';
+import './ProgressiveList.scss';
+
+export const getGeneratedText = (text, items: string[], callback) => {
```

Comment:
This can be put into a separate util file.

Reply:
Essentially this can be the structure -
- `component/progressive-list/ProgressiveList.tsx`
- `component/progressive-list/ProgressiveListItem.tsx`
- `component/progressive-list/progressive-list-utils.ts`
- `component/progressive-list/__tests__/ProgressiveList.spec.tsx`